src/table/body/table-row.component.ts

```diff
@@ -183,6 +183,10 @@ export class TableRowComponent {
 		return this.expandable ? true : null;
 	}
 
+	@HostBinding("attr.tabindex") get isAccessible() {
+		return this.enableSingleSelect && !this.showSelectionColumn ? 0 : null;
```

Comment:
I seem to still be able to focus on table rows when `enableSingleSelect` is set to `true` and `showSelectionColumn` is set to `false`. Is there any other condition I'm missing that makes a row not focusable?

Reply:
The situation you explained is the only situation that makes it true and focusable. Can you better explain what you mean? 

Possible senarios
enableSingleSelect = true + showSelectionColumn = false => 0 (focusable)
enableSingleSelect = false + showSelectionColumn = false => null (not focusable)
enableSingleSelect = true + showSelectionColumn = true => null (not focusable)
enableSingleSelect = false + showSelectionColumn = true => null (not focusable)

Reply:
@Donisius I'm also a bit confused ... Do you mean you see that behavior in the current version?

Otherwise I think we're all getting at the same thing, so we should be good to go.

Reply:
@DusanMilko @cal-smith Apologies, I thought I was able to focus on the rows in the current version so I was wondering why this was necessary.

Reply:
No worries!